code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -379,9 +509,12 @@
 	if(!message)
 		return
 	var/my_message
-	my_message = "<FONT size = 3><b>COMMAND FROM SPIDER QUEEN:</b> [message]</FONT>"
+	my_message = "<span class='alien big'><b>COMMAND FROM SPIDER QUEEN:</b> [message]</span>"

```

Comment:
It turns out this looks really shitty in game, I guess I should just stick with font 3 bold for it

Reply:
I'll fiddle a bit and make a spider span